src/main/java/core/basesyntax/service/impl/DataParseServiceImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.DataParseService;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class DataParseServiceImpl implements DataParseService {
+    private static final String COMMA_SEPARATOR = ",";
+    private static final int HEADER_INDEX = 0;
+
+    @Override
+    public List<String[]> parseData(List<String> data) {
```

Comment:
introduce model class `FruitTransaction` to store operation(along with `getOperationByCode()` method), fruit name, and amount as its fields. 
It is a bad practice to operate raw strings